package-lock.json

```diff
@@ -21447,9 +21447,9 @@
       "integrity": "sha512-60TedWOHHuqTpaZsTni9yyc+FwH4qP/Kjt6QegK7Hv1G+tP/CJ7+K1mTRsrEJkjUq55Q0a7hjHTblv+Y+u4o0Q=="
     },
     "node_modules/vue-class-component": {
-      "version": "7.2.5",
-      "resolved": "https://registry.npmjs.org/vue-class-component/-/vue-class-component-7.2.5.tgz",
-      "integrity": "sha512-0CSftHY0bDTD+4FbYkuFf6+iKDjZ4h2in2YYJDRMk5daZIjrgT9LjFHvP7Rzqy9/s1pij3zDtTSLRUjsPWMwqg==",
+      "version": "7.2.6",
```

Comment:
This is a normal patch upgrade, seen here because I totally rebuild this package lock file. It should have no effect.